edx-platform/pearson-pols-theme/lms/templates/ccx/coach_dashboard.html

```diff
@@ -120,34 +120,12 @@ <h2 class="hd hd-2" id="header-ccx-dashboard">${_(ALTERNAVITE_STRINGS.get("CCX C
 
   function setup_management_form() {
 
-    $(".member-lists-management form").on("submit", function (event) {
-      var target, action;
-      target = $(event.target);
-      if (target.serialize().indexOf('student-action') < 0) {
-        action = $('<input />', {
-          type: 'hidden',
-          name: 'student-action',
-          value: 'add'
-        });
-        target.append(action);
-      }
-    });
-
-    $(".member-lists-management form .add, .member-lists-management form .revoke").on("click", function(event) {
+    $(".member-lists-management-ccx form .revoke").on("click", function(event) {
         var target, form, action, studentId, selectedStudent;
         event.preventDefault();
         target = $(event.target);
         form = target.parents('form').first();
-        if (target.hasClass('add')) {
-          // adding a new student, add the student-action input and submit
-          action = $('<input />', {
-            type: 'hidden',
-            name: 'student-action',
-            // this is untenable, tied to a translated value.  Fix it.
-            value: 'add'
-          });
-          form.append(action).submit();
-        } else if (target.hasClass('revoke')) {
+        if (target.hasClass('revoke')) {
```

Comment:
Double quotes. @andrey-canon

Reply:
I will keep it in order to avoid differences with the original file